db/structure.sql

```diff
@@ -3403,6 +3403,59 @@ COMMENT ON COLUMN public.reporting_facility_states.appts_scheduled_32_to_62_days
 COMMENT ON COLUMN public.reporting_facility_states.appts_scheduled_more_than_62_days IS 'The total number of appointments scheduled by healthcare workers at the facility in the reporting month more than 62 days from the visit date';
 
 
+--
+-- Name: reporting_facility_states_by_genders; Type: MATERIALIZED VIEW; Schema: public; Owner: -
+--
+
+CREATE MATERIALIZED VIEW public.reporting_facility_states_by_genders AS
```

Comment:
TODO: chat through how to name these views that are doing further facility level breakdowns.